testutil/application/mocks/mocks.go

```diff
@@ -0,0 +1,3 @@
+package mocks
+
+// This file is in place to declare the package for dynamically generated structs.
```

Comment:
Just curious, is this following some conventional structure for mock packages in a cosmos-sdk app?

Reply:
I update the comment, but open to pushback if you think we should follow Cosmos for consistency.

```
// This file is in place to declare the package for dynamically generated structs.
// Note that this does not follow the Cosmos SDK pattern of committing Mocks to main.
// For example, they commit auto-generate code to main: https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/testutil/expected_keepers_mocks.go
// Documentation on how Cosmos uses mockgen can be found here: https://docs.cosmos.network/main/build/building-modules/testing#unit-tests
```

Reply:
:sunglasses: nice, ty!